index.html

```diff
@@ -121,16 +129,17 @@ <h3 class="program-title">Workshop</h3>
               alt="ignite"
             />
             <h3 class="program-title">Ignite</h3>
-            <p class="program-description">
-              Get opportunities to network with athletes around the world
+            <p class="program-descr">
+              Get opportunities to network with sports experts and
+              fellow sport lovers and athletes
             </p>
           </div>
         </div>
         <a class="program-link" href="/">SEE THE WHOLE PROGRAM</a>
         <button class="join">Join annual sports Conference 2022</button>
       </section>
```

Comment:
- [ ] OPTIONAL: I recommend aligning the content of the button in the center: 
<img width="343" alt="Screenshot 2022-03-10 at 13 29 12" src="https://user-images.githubusercontent.com/77038610/157632287-f38794f6-ad3e-4212-8e2a-98a2b06f0e48.png">